README.md

```diff
@@ -46,6 +46,13 @@ riverline_worker:
             class: Riverline\WorkerBundle\Provider\AMQP
         semaphore:
             class: Riverline\WorkerBundle\Provider\Semaphore
+        activemq:
```

Comment:
Add a note that the Statistics plugin is required. (maybe you can check if it is active ?)

Reply:
Done